assets/src/edit-story/app/media/utils/getResourceFromAttachment.js

```diff
@@ -51,6 +52,7 @@ function getResourceFromAttachment(attachment) {
   } = attachment;
   return createResource({
     mimeType,
+    createTime: date,
```

Comment:
Can we use `uploadDate` for this everywhere?

Reply:
I suggested that we use the word 'create' instead of 'upload', so that the media resource structure can be re-used (or at least follow closely) with the media 3p integrations. The word 'upload' is too general IMO, and doesn't really have any meaning for Unsplash.

WDYT?

Reply:
Works for me 👍 

Then let's do `creationDate` everywhere? The `time` part is misleading.